_src/static/sass/base/_typography.scss

```diff
@@ -48,6 +48,13 @@ p, ol, ul, dl, td, th {
     @extend %reset-copy;
 }
 
+p,
+li,
+dt,
+dd {
+    line-height: line-height(base);
```

Comment:
This should be covered by the rule above that leverages ``@extend %reset-copy;``. Right now that extend is using ``line-height: line-height(large);``. If you need to change it, I'd change it there.